tests/test_db_to_vcf.py

```diff
@@ -0,0 +1,71 @@
+"""
+test_db_to_vcf.py
+~~~~~~~~~~~~~~~~~
+"""
+import sys
+from pathlib import Path
+import importlib.util
+
+import pytest
+
+
+# some magic to import the script as a module
```

Comment:
Maybe consider a different project layout to fix this. This does not look very nice. You are testing only one script right?